components/sections/DescriptionSection.jsx

```diff
@@ -0,0 +1,129 @@
+  const [width, setWidth] = useState(null);
+  const updateWidth = () => {
+    setWidth(innerWidth);
+  };
+  useLayoutEffect(() => {
+    window.addEventListener("load", updateWidth, false);
+    return () => window.removeEventListener("load", updateWidth);
+  });
+  useLayoutEffect(() => {
+    window.addEventListener("resize", updateWidth);
+    return () => window.removeEventListener("resize", updateWidth);
+  }, []);
+
+  return (
+    <Wrapper styles="font-normal desktop:text-2xl ipad:text-2xl text-xl desktop:mt-24 desktop:mb-18 tablet:mt-14 tablet:mb-14 ipad:mt-15 mt-20 -tracking-thiner">
+      <div
+        className={`hidden ipad:grid ${
+          width < 1440 ? "tablet:grid-cols-9 " : "desktop:grid-cols-9"
+        }`}
+      >
+        <div
+          className={`hidden tablet:inline ${
+            width < 1440 ? "tablet:col-span-2" : "desktop:col-span-4"
+          }`}
+        />
+        <div
+          className={` ${
+            width < 1440 ? "tablet:col-span-7" : "desktop:col-span-5"
+          }`}
+        >
+          <div className="flex gap-8 justify-start relative tablet:max-w-47.25">
+            {DESCRIPTIONS.map((item, i) => (
+              <button
+                key={i}
+                onClick={() => setDescription(i)}
+                className={`${
+                  description === i
+                    ? "text-purple one"
+                    : "text-secondary-100 no-underline"
+                }`}
+                ref={(element) => (refs.current[i] = element)}
+              >
+                {item.header}
+                <hr
+                  style={{
+                    width: `${position.width}rem`,
+                    left: `${position.left}rem`,
+                  }}
+                  className={`absolute top-full h-0.15 inline-block bg-purple transition-all duration-150 ease-in-out`}
+                />
+              </button>
+            ))}
+          </div>
+          <div className="my-6 relative">
+            <div>
+              {DESCRIPTIONS[description].description}
+              <span className="ipad:inline-block underline hover:no-underline hover:duration-300 ease-in-out desktop:hidden">
+                <Button style="hover:no-underline">Learn more</Button>
+              </span>
+            </div>
+
+            <p className="my-6">{DESCRIPTIONS[description].end}</p>
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col ipad:hidden gap-4 justify-start w-full">
+        {DESCRIPTIONS.map((item, i) => (
+          <div key={i}>
+            <button
+              onClick={() => {
+                setDescription(i);
+                setShow(!show);
+              }}
+              className={`${
+                description === i && show
+                  ? " text-purple border-b border-purple py-1 stroke-purple"
+                  : "text-black border-b border-black stroke-black"
+              } flex justify-between  w-full`}
```

Comment:
Always put conditional styles at the end
```suggestion
              className={`flex justify-between border-b  w-full ${
                description === i && show
                  ? " text-purple border-purple py-1 stroke-purple"
                  : "text-black border-black stroke-black"
              }`}
```